cloudinit/config/cc_landscape.py

```diff
@@ -128,14 +140,16 @@ def handle(name: str, cfg: Config, cloud: Cloud, args: list) -> None:
         LSC_CLIENT_CFG_FILE,
         ls_cloudcfg,
     ]
-    merged = merge_together(merge_data)
-    contents = BytesIO()
-    merged.write(contents)
-
-    util.ensure_dir(os.path.dirname(LSC_CLIENT_CFG_FILE))
-    util.write_file(LSC_CLIENT_CFG_FILE, contents.getvalue())
-    LOG.debug("Wrote landscape config file to %s", LSC_CLIENT_CFG_FILE)
+    cmd_params = []
+    for k, v in sorted(merge_together(merge_data)["client"].items()):
+        if re.search(r"(\"|\s)", v):
+            cmd_params.append(
+                f"--{k.replace('_', '-')}=\"" + v.replace('"', '"') + '"'
```

Comment:
I believe the intention is to quote the inner double quotes, right ?

```suggestion
                f"--{k.replace('_', '-')}=\"" + v.replace('"', '\\"') + '"'
```